### ISSUE TITLE
PaP Camo Overlay (IDEA)

### ISSUE THREAD

**Issue by sh4d0w59** [Zet0rz/nzombies#582]
Instead of a direct texture replacement, can we have the option between the current system and an "overlay" system.  What I mean by overlay is taking a slightly scaled up viewmodel (scale ~1.05) and overlaying it onto the actual viewmodel.  With this system you could use transparent animated materials such as "models/props_combine/portalball001_sheet.vmt" to get an effect similar to the gun in this ( https://vignette.wikia.nocookie.net/callofduty/images/2/2b/KN-44_Gunsmith_model_Dark_Matter_Camouflage_BO3.gif/revision/latest?cb=20160122110023 ) picture.  Notice how the original texture is still under the animated texture.

**Comment by DjNathann** [comment]
(sorry for being rude)"edited"

**Comment by sh4d0w59** [comment]
@DjNathann No need to be a dickhead.  Im simply suggesting an idea.  I dont expect it to be implemented right away or at all.  Why do you think the word "IDEA" is in the title?  Also, how would I know if Zet0rz has time or not?  To my knowledge he has no social media outlet that shows what he's doing at any given time.

**Comment by minin43** [comment]
I agree with sh4d0w here (dumb name, by the way). It was just a suggestion, there were no expectations made by saying it, or in the suggestion itself.

Lay off the sarcasm, Nathan.

**Comment by Ethorbit** [comment]
It's funny when DJ tells someone they don't know how to code. A bit ironic.
I don't think the issues section is a suitable spot for ideas.

**Comment by minin43** [comment]
The issues section is the only spot for ideas.

**Comment by Ethorbit** [comment]
No, the issues section is for issues and ideas on how to fix those issues. This topic is not related to that. What issue is this about? I forgot.

And if you mean this is the only place to post the ideas at, you are wrong. You can contact Zet0rz elsewhere.

**Comment by minin43** [comment]
You're quite defensive and aggressive for only having recently started participating in these discussions. What's the deal?

No, there's no communal area for fans of the gamemode to make and discuss suggestions, besides this area.

It's been used for suggestions for the past 2 years. Zet0r is fine with suggestions being present here. Why do you have a stick up your ass about it?

**Comment by Ethorbit** [comment]
Sorry if I hurt your feelings.

**Comment by sh4d0w59** [comment]
There arent any places to post ideas on github by default (No dedicated tab) and like @minin43 said, Zet0r doesnt care about ideas in the issues tab.  The best I can do to separate ideas from issues on my side is to mark them the the tag (IDEA) at the end of the title.  I think it gets the job done though because I haven't got any complaints in the past from posting ideas.  If Zet0r wants the posts taken down, he can do it himself or I'll gladly take the posts down if he asks.  /end rant

**Comment by minin43** [comment]
>Sorry if I hurt your feelings.

See, we're gonna have a problem here if you're gonna continue with the passive-aggressive, for-lack-of-a-better-word bullshit responses. It's very clear my feelings aren't hurt. Why would they be? I'm just trying to resolve a problem in a courteous fashion, and you're being a condescending prick in your responses. Let's knock it off, shall we?

**Comment by Ethorbit** [comment]
Look, I genuinely apologize for pissing you off. You're saying I'm trying to be all defensive, but just look at your posts. Anyway I will be muting this discussion now as I'm tired of the constant github notifications. 

How was I supposed to know that Zet0rz was okay with it? (Never said he wasn't) all I said was what the Issues thread means, because what else could it possibly mean to someone who wasn't here a year ago? 

Don't actually answer those, I won't be able to see them.

We're done here.

**Comment by minin43** [comment]
Ahh, the classic 

>well I don't want to try and actually justify the way I acted or the things I said so I'm just gonna throw together a dumb apology and pretend to not hear anything else anyone says

Honestly, a true classic. 'Course who am I to say "knock it off" when I continue it?

@sh4d0w59 Have you tried messaging Zet0r directly lately? He's seem to gone quiet and hasn't updated in awhile.

**Comment by sh4d0w59** [comment]
I know you wont be able to see this @Ethorbit but no hard feelings.  I dont want to ruin a potential friendship / aquaintanceship over nonsense.  Have a nice day or night.

**Comment by sh4d0w59** [comment]
@minin43 I havent.  I think he may be taking a break.  I dont want to disturb him if hes taking time to decompress.

**Comment by Zet0rz** [comment]
Posting suggestions or ideas is fine in the Issues tab, I just need to properly tag them like I have now lol.
I do read most of the the notifications I get, main reason for being so inactive is actually that I read them at times where it's inconvenient to reply (checking through emails before heading to sleep). I normally have github open when I develop code, but lately work has focused on modelling and not the development of the actual gamemode code (not to mention it having been real slow too, mainly because of my side projects and my youtube and twitch channels).

I don't know when I will finish with the models, they are pretty much done but I need to modify the barricade animations which has shown itself to be a big task for someone who is not a modeller like me lol.

Anyway, for this idea itself I'll look into seeing if it's possible to overlay instead of replace. If it is, I'll probably implement it when I also implement the skin system (allowing you to set camos, round change music, zombie skins etc).
